metadata/upload.go

```diff
@@ -28,6 +29,7 @@ var (
 	projectID       = os.Getenv("GCP_PROJECT")
 	datasetID       = os.Getenv("DATASET_ID")
 	tableID         = getEnv("TABLE_ID", "metadata")
+	gcsFetchBufferSize, _ = strconv.Atoi(os.Getenv("GCS_QUERY_THROTTLE"))
```

Comment:
```suggestion
	gcsFetchBufferSize, err = strconv.Atoi(getEnv("GCS_QUERY_THROTTLE", "500"))
	if err != nil {
	   return
	}
```
This block should be moved to `init()` function, to be able to use error handling.